R/tar_insuvimeh.R

```diff
@@ -0,0 +1,123 @@
+catalogue_insuvimeh_files <-  function(gdb){
+  fps <- list.files(gdb,
+                    full.names =T,
+                    recursive = T,
+                    pattern = "\\d{4}.nc$")
+  fps_w_parents <- list.files(gdb,
+                              recursive = T,
+                              pattern = "\\d{4}.nc$")
```

Comment:
hmmm what am i trying to do here? i think i do want the pattern I provided and for the metadata catalogue tibble, i wanted the full file path, file name, and the parent folder name. This was due to some custom file storage convention that the insuvimeh used when sharing the data. 

i think its working correctly, but could probably have been dealt with more cleanly with some file path tools? maybe `{fs}` ?

Reply:
I just mean, in the parent function you specify `pattern` as a parameter with the default being `\\d{4}.nc$`, so don't you want `pattern = pattern` in `list.files()`? Otherwise the parameter does nothing.

Reply:
ha i think there is no parent function, but my eyes just played a trick on me and made me think there was as well. Nonetheless I kind of like the adea of making it a parameter so added it here [2ac211c](https://github.com/OCHA-DAP/ds-aa-lac-dry-corridor/pull/12/commits/2ac211c2fbef6d777945776bbbc26b1dac0502ec)